e2e-tests/native/test/helpers/Gestures.ts

```diff
@@ -0,0 +1,179 @@
+// For reference https://github.com/webdriverio/appium-boilerplate/blob/main/tests/helpers/Gestures.ts
```

Comment:
This file is mainly interesting because it is a lot better commentated an structured than the previous implementation and it provides this helper `checkIfDisplayedWithSwipeUp`